failed to resolve: use of undeclared type or module `log`

I'm seeing this error after #537 is merged. Probably, `log` crate should be moved to dependecies instead of dev-dependencies...?

```
error[E0433]: failed to resolve: use of undeclared type or module `log`
   --> sphere/src/main.rs:430:42
    |
430 |             &device.create_shader_module(wgpu::include_spirv!("shaders/shader.vert.spv")),
    |                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `log`
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared type or module `log`
   --> sphere/src/main.rs:431:42
    |
431 |             &device.create_shader_module(wgpu::include_spirv!("shaders/shader.frag.spv")),
    |                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `log`
    |
    = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/src/macros.rs
+++ b/src/macros.rs
@@ -48,7 +48,7 @@ fn test_vertex_attr_array() {
 macro_rules! include_spirv {
     ($($token:tt)*) => {
         {
-            log::info!("including '{}'", $($token)*);
+            //log::info!("including '{}'", $($token)*);
             $crate::util::make_spirv(include_bytes!($($token)*))
         }
     };

